src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
+$mainColor: #060b35;
+$otherColor: #616070;
+$buttonColor: #00acdc;
```

Comment:
you can create a separate `variables.scss` file and store your colors there. (just import this file into `index.scss` using `@import`)